avaframe/out3Plot/plotUtils.py

```diff
@@ -209,24 +209,33 @@ def saveAndOrPlot(cfgPath, cfgFlags, outFileName, fig):
     return
 
 
-def addColorBar(im, ax2, ticks, myUnit):
+def addColorBar(im, ax2, ticks, myUnit, title=''):
     '''
     Adds, styles and labels a colorbar to the given image and axes
     '''
     cbar = ax2.figure.colorbar(im, ax=ax2, ticks=ticks)
     cbar.outline.set_visible(False)
     cbar.ax.set_title('[' + myUnit + ']')
+    if title != '':
+        cbar.set_label(title)
 
 
 def putAvaNameOnPlot(ax, avaDir):
     '''
-    Puts the date and avalanche name in the lower left corner of the given
+    Puts the date and avalanche name (or a list of ava names) in the lower left corner of the given
     matplotlib axes
     '''
-    avaName = pathlib.PurePath(avaDir).name
 
-    infoText = datetime.datetime.now().strftime("%d.%m.%y") + \
+    if isinstance(avaDir, str):
```

Comment:
Please add a short comment what this if statement switches

Reply:
Added a comment